packages/theme-chalk/src/statistic.scss

```diff
@@ -0,0 +1,52 @@
+@use 'mixins/mixins' as *;
+@use 'mixins/var' as *;
+@use 'common/var' as *;
+
+@include b(statistic) {
+  @include set-component-css-var('statistic', $statistic);
+}
+
+@include b(statistic) {
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+  font-variant: tabular-nums;
+  line-height: 1.5715;
+  list-style: none;
+  font-feature-settings: 'tnum';
+
+  .head {
+    padding: 0 5px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    letter-spacing: -0.01px;
```

Comment:
Why need `letter-spacing: -0.01px;`, What's the purpose?

Reply:
I found this in the css in the draft and added it

Reply:
You could modify and remove any code just make sense.

Reply:
OK, I'll adjust it👌